flake.nix

```diff
@@ -45,6 +45,14 @@
         }).nixos.manual.x86_64-linux;
       };
 
+      # The "legacy" in `legacyPackages` doesn't imply that the packages exposed
+      # through this attribute are "legacy" packages. Instead, `legacyPackages`
+      # is used here as a substitute for `packages` because enumerating the
+      # entire list of packages in Nixpkgs is quite computationally expensive
+      # due to the sheer number of packages, which in turn makes an operation
```

Comment:
More accurately, it will make `nix flake show nixpkgs --legacy` slow, as it recursively enumerates all the packages; the attribute is named as such so it can be special-cased to not do the "normal" thing for flakes.

Reply:
Ah, good point. My intended meaning was that using `packages` would make `nix flake show` unusably slow, which is not clear here.